mathics/builtin/__init__.py

```diff
@@ -37,6 +38,50 @@
 )
 
 
+def is_builtin(var):
+    if var == Builtin:
```

Comment:
I'm not sure, but it seems this should be something like is or isinstance

Reply:
Indeed, it should be `is`.